File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p03703/p03703_num2_parsed.py
import sys

N, K = map(int, sys.stdin.readline().split())
a = [int(sys.stdin.readline()) for _ in range(N)]

count = 0
for i in range(N):
    for j in range(i, N):
        subsequence_sum = sum(a[i:j+1])
        subsequence_len = j - i + 1
        if subsequence_sum / subsequence_len >= K:
            count += 1

print(count)

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p03703/p03703_num2_parsed.cpp
#include <iostream>
#include <vector>

using namespace std;

int main() {
    int N, K;
    cin >> N >> K;

    vector<int> a(N);
    for (int i = 0; i < N; i++) {
        cin >> a[i];
    }

    int count = 0;
    for (int l = 0; l < N; l++) {
        int sum = 0;
        for (int r = l; r < N; r++) {
            sum += a[r];
            if (sum / (r - l + 1) >= K) {
                count++;
            }
        }
    }

    cout << count << endl;

    return 0;
}